### ISSUE TITLE
DiffuseHigh for 2K+ resolution with any model

### ISSUE THREAD

**Issue by dathide** [comfyanonymous/ComfyUI#4891]
### Feature Idea

"DiffuseHigh enables the pre-trained text-to-image diffusion models (SDXL in this figure) to generate higher-resolution images than the originally trained resolution, e.g., 4×, 16×, without any training or fine-tuning."
https://yhyun225.github.io/DiffuseHigh

**Comment by blepping** [comment]
@dathide if you're still looking, i made a version for ComfyUI. currently lightly tested/alpha status.

Repo link: https://github.com/blepping/comfyui_jankdiffusehigh

has a few possible improvements:

* you can use any sampler/schedule for the reference or upscale iteration steps.
* you can use an upscale model instead of just scaling the image.
* possible to apply guidance via the latent rather than image, saving a VAE round trip (not sure it's better, it's definitely much faster though).
* possible to apply the guidance during guidance steps at less than 100% strength, also can fade it out over time.
* possible to use contrast adaptive sharpening rather than gaussian-blur based sharpening.

**Comment by dathide** [comment]
Yes, I was definitely still looking. Thank you, I installed it and gave it a star. I've done a couple tests with good results besides some color issues that I was also having with other upscaling methods.

**Comment by blepping** [comment]
thanks for the feedback! glad to hear it seems to be working for you. if you run into any issues/have questions/etc please feel free to open an issue or start a discussion in that repo.
